src/main/java/lotto/step2/domain/LottoStore.java

```diff
@@ -0,0 +1,40 @@
+package lotto.step2.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoStore {
+    private static final int MIN_LOTTO_NUMBER = 1;
+    private static final int MAX_LOTTO_NUMBER = 45;
+    private static final int LOTTO_SIZE = 6;
+
+    private static List<LottoNumber> allLottoNumbers = new ArrayList<>();
+
+    static {
+        IntStream.rangeClosed(MIN_LOTTO_NUMBER, MAX_LOTTO_NUMBER)
+                .forEach(number -> allLottoNumbers.add(new LottoNumber(number)));
+    }
```

Comment:
```suggestion
    static {
        for (int number = MIN_LOTTO_NUMBER; number <= MAX_LOTTO_NUMBER; number++) {
            allLottoNumbers.add(new LottoNumber(number));
        }
    }
```

단순한 forEach 문 만을 사용할 거라면, 기존의 for loop 와 비교하였을 때,
가독성 측면에서도, 성능상으로도 아무런 장점이 없습니다.
IntStream 을 reduce 하거나 map 을 하는게 아니라, forEach 만 할거라면 기존의 for 문을 활용해주세요.

Reply:
람다식을 쓰면 좋을거라고 생각했는데 성능상 안 좋을 때도 있군요!

Reply:
람다식은 무조건 일반적인 for loop 보다 성능이 안 좋다고 생각하시면 됩니다.
그럼에도 쓰는 이유는 강력한 Stream 의 기능을 쓰기 위해서지요.
위와 같은 경우에는 Stream 의 기능을 적극적으로 사용하고 있지 않은 상황인지라 언급해보았습니다.